### DISCUSSION TITLE
How do be pass Option values using ts-sdk ??

### DISCUSSION THREAD

**Discussion by ajaythxkur** [aptos-labs/aptos-developer-discussions#375] [Questions] [answered]
Suppose i have a function that accepts `Option<String>` parameter. How is it sent from client using sdk? One with string and one with nothing...

**Comment by ajaythxkur** [comment]
Basically this
0x1::option::Option<0x1::string::String>

**Reply by Alexzander-Stone** [reply, marked as answer]
Here's a [previous discussion answer](https://github.com/aptos-labs/aptos-developer-discussions/discussions/61#discussioncomment-8777351) that covers Options in the TS SDK.
